docs/api-reference/provenance-database.md

```diff
@@ -0,0 +1,98 @@
+- `end_xid`: The transaction ID that deleted the record/row, or superseded the record with a new version. Each delete or update of the record in your application database does not delete the old record in the provenance database, instead, DBOS updates the `end_xid` field of the latest record (with `end_xid` set to "infinity" `9223372036854775807`) to the transaction that deleted it. You can use this column to check which transaction deleted or updated this record.
+- `begin_seq`: The insert/update/delete SQL statement sequence number within the transaction that added this record. This field is used by the Time Travel Debugger to replay transactions.
+- `end_seq`: The insert/update/delete SQL statement sequence number within the transaction that deleted this record. This field is used by the Time Travel Debugger to replay transactions.
+
+To make sure that you can access previous data, DBOS provenance tables are append-only, and each provenance table includes all previous columns of this table.
+This means if you delete a column from a table, you are still able to view them in the provenance database.
+DBOS Cloud periodically garbage collects and compacts provenance tables according to your [data retention period](https://www.dbos.dev/pricing).
+
+For example, for a deployed ["Hello, Database" quickstart](../getting-started/quickstart) application, the `dbos_hello` table looks like this after we send several greetings to Mike:
+
+```sql
+hello=> select * from dbos_hello;
+ name | greet_count
+------+-------------
+ Mike |           6
+```
+
+Then in the provenance database, the `dbos_hello` table is extended with four additional columns and records all versions of this record:
+```sql
+hello_dbos_prov=> select * from dbos_hello;
+ name | greet_count | begin_xid | begin_seq |       end_xid       | end_seq
+------+-------------+-----------+-----------+---------------------+---------
+ Mike |           1 |     24818 |         1 |               24824 |       1
+ Mike |           2 |     24824 |         1 |               24826 |       1
+ Mike |           3 |     24826 |         1 |               24832 |       1
+ Mike |           4 |     24832 |         1 |               24834 |       1
+ Mike |           5 |     24834 |         1 |               24841 |       1
+ Mike |           6 |     24841 |         1 | 9223372036854775807 |       0
+```
+
+You can see that transaction `begin_xid=24818` initially set Mike's `greet_count` to 1, and this record was updated by transaction `end_xid=24824` which updated `greet_count` from 1 to 2.
+The latest version with `greet_count = 6` was created by transaction `24841`.
+
+## DBOS System Tables
+The provenance also stores a replica of the DBOS system tables under the `dbos` schema which record each function/workflow execution.
```

Comment:
A copy of the DBOS system tables (under the `dbos` schema), which record each function/workflow execution, is also maintained in the provenance database.